packages/blockchain-wallet-v4/src/redux/common/btc/selectors.js

```diff
@@ -113,12 +115,16 @@ export const getWalletTransactions = state => {
   // [Remote([tx])] == [Page] == Pages
   const getDescription = (hash, to) => TXNotes.selectNote(hash, Wallet.selectTxNotes(wallet)) || getAddressLabel(to, state).getOrElse('')
 
+  const getPartnerLabel = hash => getShapeshiftTxHashMatch(state, hash) || getBuySellTxHashMatch(state, hash)
+
   const pages = getTransactions(state)
-  const metadata = getMetadata(state)
+  // Remote(metadata)
+  getMetadata(state)
```

Comment:
I think we at least have to do this

Reply:
@salomegeo I tested the address labels and they still work even if we remove metadata parameter in ProcessTxs